addons/website/static/src/js/tours/tour_utils.js

```diff
@@ -13,6 +13,18 @@ function addMedia(position = "right") {
         run: "click",
     };
 }
+function assertCssVariable(variableName, variableValue, trigger = "iframe body") {
```

Comment:
Absolutely nitpicking one: don't we usually use double quote only for string that will be displayed, and single string for technical stuff?